docs/migration/database.md

```diff
@@ -0,0 +1,75 @@
+# Database Migration Guide
+
+Migrating databases between pods is a straightforward process with 5 steps:
+1. Dump the databases to a dump file.
+2. Copy the dump file out of the old pod.
```

Comment:
old postgres?

Reply:
This no longer applies. I have changed the docs.